.xtbrc

```diff
@@ -14,7 +14,7 @@ $opt
    maxcycle=0
    maxdispl=2.000000000000000
    hlow=.5000000000000000E-02
-   s6=20.00000000000000
+   s6=100.00000000000000
```

Comment:
Sure that this is a save change for all methods?

Reply:
s6=70